test/remote/gateways/remote_trans_first_test.rb

```diff
@@ -70,12 +70,12 @@ def test_failed_purchase
     assert_equal 'Insufficient funds', response.message
   end
 
-  def test_successful_refund
```

Comment:
@nfarve we'll actually want to keep this test even though TransFirst has a long delay between a purchase and the transaction fully settling. In the future someone could want to do a remote test for refund and just manually run a purchase and wait to run the refund test till the next day. Although we should have a remote `void` test so we should add that one while we're at it! I'm wondering why this never had a `void` test to begin with 🤔  

Also, it could be good to add a partial refund remote test- [for example EBANX](https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_ebanx_test.rb#L90-L97)- and this can follow the same test pattern as a full refund test where the "tester" just waits until the next day to exercise the test.